### DISCUSSION TITLE
i update to webpack5，and got an error as follows，anyone could helps me,thanks a lot

### DISCUSSION THREAD

**Discussion by Henrycheheng** [webpack/webpack#18717] [Q&A]
node_modules/webpack/lib/DefinePlugin.js:371                                    (compilation.valueCacheVersions.get(VALUE_DEP_MAIN)) || ""                                                      ^
TypeError: Cannot read properties of undefined (reading 'get')

**Comment by alexander-akait** [comment]
Can you create repreducible test repo?
